packages/web3-providers/src/entry.ts

```diff
@@ -197,3 +197,4 @@ export const RSS3 = new RSS3API()
 
 // Storage
 export * from './Storage/index.js'
+export * from './Lens/utils.js'
```

Comment:
We should avoid exporting biz helpers from `@masknet/web3-providers`. Only if they are publicly shared. This entry is for exporting APIs. Use entry-helpers stead.

Reply:
By the way, what's the difference between `isLens` in `packages/web3-shared/evm/src/helpers/resolver.ts`.